utils_test.go

```
@@ -0,0 +1,55 @@
+	}
+
+	var body bytes.Buffer
+	w := multipart.NewWriter(&body)
+	if err := addFile(w, "file", path); err != nil {
+		t.Fatalf("addFile returned error: %v", err)
+	}
+	if err := w.Close(); err != nil {
+		t.Fatal(err)
+	}
+
+	req, err := http.NewRequest(http.MethodPost, "http://example.invalid", &body)
```

golangci-lint (2.13.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)